src/examples/install-cli.yml

```diff
@@ -1,10 +1,16 @@
 description: >
-  This example shows how to install the lacework-cli
+  This example shows how to install the lacework-cli.
+
+  For more information about the Lacework CLI visit:
+    - https://github.com/lacework/go-sdk/wiki/CLI-Documentation
 usage:
   version: 2.1
   orbs:
     lacework: lacework/lacework@1.2.3
```

Comment:
replace the placeholder version number with x.y to prevent users from copying the example directly, and encourage users to accept patch updated by only setting the major and minor versions.